patterns/gitleaks/8.27.0/99-testing.toml

```diff
@@ -169,3 +169,39 @@
   entropy = 3
   keywords = ['client']
   tags = ['type:secret', 'group:leaktk-testing']
+
+[[rules]]
+  # TODO: merge with mC5kOxQ6kn4 after testing
+  id = 'B09tH8vORCw'
+  description = 'Generic Secret'
+  # This one looks for file names like
+  # - token
+  # - password
+  path = '''(?i)(?:^|\/)(?:password|secret|token)$'''
+  regex = '''[\S]{8,}'''
```

Comment:
Are you sure you want to use `\S` instead of something like `[[:graph:]]`? In my experience, it can match surprising things (e.g., invalid bytes).